src/pages/success-stories/studiodao.page.tsx

```diff
@@ -0,0 +1,113 @@
+import { Trans, t } from '@lingui/macro'
+import { CaseStudiesConfigurationProps } from 'components/CaseStudies/CaseStudiesConfiguration'
+import { CaseStudiesHeader } from 'components/CaseStudies/CaseStudiesHeader'
+import { CaseStudyContentContainer } from 'components/CaseStudies/CaseStudyContentContainer'
+import ExternalLink from 'components/ExternalLink'
+import SectionHeader from 'components/SectionHeader'
+import { AppWrapper } from 'components/common'
+import { CASE_STUDY_PROJECTS } from 'constants/successStoryProjects'
+import Image from 'next/image'
+
+export default function StudioDAOPage() {
+  const studioDAOSuccessStoryProject = CASE_STUDY_PROJECTS[3]
+
+  const fundingConfig: CaseStudiesConfigurationProps = {
```

Comment:
these config constants can exist _outside_ of the component, for slightly better performance.

Reply:
Moved to `constants/successStoryProjects`